src/components/RichText.ts

```diff
@@ -20,27 +20,35 @@ export interface CommonRichTextProps {
     customOptions: { option: string }[];
     minNumberOfLines: number;
     maxNumberOfLines: number;
+    recreate?: boolean;
 }
 
 export interface RichTextProps extends CommonRichTextProps {
     className?: string;
-    onChange?: (value: string) => void;
+    onChange?: (value?: string) => void;
+    onBlur?: () => void;
+    update: boolean;
```

Comment:
I don't understand `update` is there a better name, and why is it needed, it makes the code not react like and feels hackish (what is probably is, and it that case i expect some comment in the code)

Reply:
Renamed to updateEditor and comment added as well.